packages/toast/src/Toast.ts

```diff
@@ -122,24 +122,26 @@ export class Toast extends FocusVisiblePolyfillMixin(SpectrumElement) {
         switch (variant) {
             case 'info':
                 return html`
-                    <sp-icon-info
-                        label="Information"
-                        class="type"
-                    ></sp-icon-info>
+                    <slot name="icon" class="type">
```

Comment:
If you wrap the call to `renderIcon()` with this instead you can do this once while also being able to add an icon when there is no variant.